CHANGELOG.asciidoc

```diff
@@ -155,28 +155,12 @@ https://github.com/elastic/beats/compare/v7.4.1...v7.5.0[View commits]
 
 ==== Deprecated
 
-*Affecting all Beats*
-
-*Filebeat*
-
-*Heartbeat*
-
-*Journalbeat*
-
 *Metricbeat*
 
 - `kubernetes.container.id` field for `state_container` is deprecated in favour of ECS `container.id` and `container.runtime`. {pull}13884[13884]
 
-*Packetbeat*
-
-*Winlogbeat*
-
-*Functionbeat*
-
 ==== Known Issue
```

Comment:
Delete this heading (the empty section is causing the doc build to fail).

Reply:
I removed this section.  Good catch.